### PR TITLE
Cast js-defined variables to string

### LINKED ISSUE #83: TypeError in './lib/resolve-value.js' when the value of css-variable is not of string type.
Hello! 

I use your great package with [this postcss.config](https://github.com/shonie/postcss-css-variables-bug-reproduce/blob/master/postcss.config.js). 

I pass plain javascript object with variables to option [**variables**](https://github.com/MadLittleMods/postcss-css-variables#variables-default-). But if any variable is set to not-string value build process crashes with this TypeError.

screen shot 2018-09-07 at 1 49 37 pm

I've created reproduce repo [here](https://github.com/shonie/postcss-css-variables-bug-reproduce). 

You can do:
```sh
git clone https://github.com/shonie/postcss-css-variables-bug-reproduce.git

npm i

npm start
```

Can i create PR for fixing this?

### PR DESCRIPTION
**Pull request by shonie**
Fix #83

Cast js-defined variables to string

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ var MOCK_JS_VARIABLES =  {
 	'js-defined-no-prefix': '#ff0000'
 };
 
+var INVALID_JS_VARIABLES = {
+	'number-value': 50,
+	'zero-value': 0,
+	'null-value': null,
+	'undefined-value': undefined,
+	'object-value-passed-by-mistake': {}
+};
```
Comment (MadLittleMods) [inline test/test.js]: We should add a `Boolean`(`true`/`false`)

File: test/test.js [hunk 2]
```diff
@@ -22,6 +22,14 @@ var MOCK_JS_VARIABLES =  {
 	'js-defined-no-prefix': '#ff0000'
 };
 
+var INVALID_JS_VARIABLES = {
```
Comment (MadLittleMods) [inline test/test.js]: Perhaps rename to `NON_STRING_VARIABLES`

File: test/test.js (hunk 1 above)
Comment (shonie) [reply test/test.js]: Agree, done in shonie@2c651e8f3df1b835aa023375d74a2306c9bfef96

File: test/test.js (hunk 2 above)
Comment (shonie) [reply test/test.js]: Agree too. It is shonie@2c651e8f3df1b835aa023375d74a2306c9bfef96 too. 

Can i squash commits now? 

Thank you for review!

Comment (MadLittleMods) [reply test/test.js]: Squash welcome 🙂

Comment (shonie) [reply test/test.js]: Done!

**Comment by shonie** [comment]
You welcome, @MadLittleMods! 🙂

**Pull request merged**